webapp/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "webapp",
```

Comment:
Why add a new package.json instead of adding relevant parts to the existing one?

Reply:
Answered this below